main.cpp

```diff
@@ -1859,9 +1861,18 @@ bool load_command::execute(device_map &devices) {
                     file_buf.insert(file_buf.begin(), read_range.from - aligned_range.from, 0);
                     file_buf.insert(file_buf.end(), aligned_range.to - read_range.to, 0);
                     assert(file_buf.size() == FLASH_SECTOR_ERASE_SIZE);
+
                     con.exit_xip();
```

Comment:
Looking at the "verify" part of the code (in the `if (settings.load.verify)` block), it seems this `con.exit_xip();` could also possibly be moved inside the `if (!skip)` block?

Reply:
Done.